### PR TITLE
fix #408

### LINKED ISSUE #408: app crashes on attempt to reopen session details after swipe up and clean
Assigned to @kgmyshin 

## Overview (Required)

app crashes on attempt to reopen session details after swipe up and clean

## Links

* https://twitter.com/ticktackmobile/status/839120043824697344
* https://twitter.com/ticktackmobile/status/839122351048404994

### PR DESCRIPTION
**Pull request by kgmyshin**
## Issue
- #408 

## Overview (Required)
- clean callbacks
- make it same each destroy-called-timing

## Links
- #408 

## Screenshot
-

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java [hunk 1]
```diff
@@ -52,6 +52,13 @@ public void onAttach(Context context) {
     }
 
     @Override
+    public void onDetach() {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java]: 🙇 👍

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java [hunk 2]
```diff
@@ -52,6 +52,13 @@ public void onAttach(Context context) {
...
+        viewModel.setCallback(null);
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java]: 🙋
I guess the callback is set null in `viewModel#destroy()` like this.

```java
public void destroy() {
    this.callback = null;
}
```

Is it better to call `setCallback(null)` here? 🤔

Comment (kgmyshin) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java]: @konifar i did it 🎅

**Comment by konifar** [comment]
Cool! Perfect :100:

**Pull request merged**